scripts/test-data/profile-l40s-x8.yaml

```diff
@@ -0,0 +1,136 @@
+  model: ~/.cache/instructlab/models/mistralai/Mixtral-8x7B-Instruct-v0.1
+  # Number of CPU cores to use for generation
+  num_cpus: 10
+  # Directory where generated datasets are stored
+  output_dir: ~/.local/share/instructlab/datasets
+  # Directory where pipeline config files are stored
+  pipeline: full
+  # The total number of instructions to be generated
+  sdg_scale_factor: 30
+  seed_file: ~/.local/share/instructlab/internal/seed_tasks.json
+  # Branch of taxonomy used to calculate diff against
+  taxonomy_base: empty
+  # Directory where taxonomy is stored and accessed from
+  taxonomy_path: ~/.local/share/instructlab/taxonomy
+  # Teacher model specific settings
+  teacher:
+    # Serving backend to use to host the teacher model
+    backend: vllm
+    # Path to teacher model that will be used to synthetically generate training data
+    model_path: ~/.cache/instructlab/models/mistralai/Mixtral-8x7B-Instruct-v0.1
+    # vLLM serving settings
+    vllm:
+      # number of GPUs to allocate to vLLM
+      gpus: 4
+      # the family of model being served - used to determine the appropriate chat template
+      llm_family: 'mixtral'
+serve:
+  # Serving backend to use to host the model
+  backend: vllm
+  # Chat template to supply to the served model. Possible values:
+  #   - Custom chat template string
+  #   - Auto: Uses default for serving backend
+  chat_template: auto
+  # Llamacpp serving settings
+  llama_cpp:
+    # number of model layers to offload to GPU
+    # -1 means all
+    gpu_layers: -1
+    # the family of model being served - used to determine the appropriate chat template
+    llm_family: ''
+    # maximum number of tokens that can be processed by the model
+    max_ctx_size: 4096
+  # Path to model that will be served for inference
+  model_path: ~/.cache/instructlab/models/instructlab/granite-7b-lab
+  # vLLM serving settings
+  vllm:
+    gpus: 4
+    # the family of model being served - used to determine the appropriate chat template
+    llm_family: ''
+    # additional arguments to be supplied directly to vLLM
+    vllm_args: ["--tensor-parallel-size", "4"]
+train:
+  additional_args:
+    warmup_steps: 25
+    learning_rate: 2e-5
+    lora_dropout: 0.1
+    lora_alpha: 32
+    deepspeed_cpu_offload_optimizer_pin_memory: true
+    deepspeed_cpu_oddload_optimizer_ratio: 1
+  ckpt_output_dir: checkpoints
+  data_output_dir: train-output
+  data_path: ./taxonomy_data
+  deepspeed_cpu_offload_optimizer: true
+  effective_batch_size: 128
+  lora_quantize_dtype: null
+  lora_rank: 0
+  max_batch_len: 30000
+  max_seq_len: 4096
+  model_path: ~/.cache/instructlab/models/instructlab/granite-7b-lab
+  num_epochs: 4
+  save_samples: 1000
+  is_padding_free: true
+  nproc_per_node: 8
+  phased_phase1_effective_batch_size: 32
+  phased_phase1_num_epochs: 4
+  phased_phase1_samples_per_save: 0
+  phased_phase2_effective_batch_size: 32
+  phased_phase2_num_epochs: 4
```

Comment:
same here

Reply:
Will increase to 7 after 4 is proven to work